cartoframes/datasets.py

```diff
@@ -0,0 +1,285 @@
+import binascii as ba
```

Comment:
There is currently a `cartoframes.analysis.Table` class that should be deprecated in favor of this one. There are fairly minimal code changes to replace that with this.

Reply:
https://github.com/CartoDB/cartoframes/issues/587